dvol/test_dvol.py

```diff
@@ -71,15 +75,64 @@ def test_list_empty_volumes(self):
         dvol.parseOptions(["-p", self.tmpdir.path, "list"])
         self.assertEqual(dvol.voluminous.getOutput(), ["  VOLUME   BRANCH   CONTAINERS "])
 
-    def test_list_multi_volumes(self):
+    # TODO Bring in jml's path_segments strategy.
+    # XXX Fix more boring wrapping output bugs (112 was too large, here).
+    @given(
+            vol_a=text(alphabet=letters[:26], min_size=1, max_size=112/2),
+            vol_b=text(alphabet=letters[:26], min_size=1, max_size=112/2),
```

Comment:
This 112 value appears in a few places. Might be a good idea to give it a 'constant'.